libs/langgraph/langgraph/pregel/utils.py

```diff
@@ -17,3 +23,29 @@ def get_new_channel_versions(
         new_versions = current_versions
 
     return new_versions
+
+
+def find_subgraph_pregel(candidate: Runnable) -> Optional[Runnable]:
+    candidates: list[Runnable] = [candidate]
+
+    for c in candidates:
+        # Cannot do isinstance(c, Pregel) due to circular imports
```

Comment:
you can import Pregel inside the function?

Reply:
ah nice